core/src/test/java/org/apache/iceberg/LocalTableOperations.java

```diff
@@ -27,13 +27,13 @@
 import org.apache.iceberg.relocated.com.google.common.collect.Maps;
 import org.junit.rules.TemporaryFolder;
 
-class LocalTableOperations implements TableOperations {
+public class LocalTableOperations implements TableOperations {
```

Comment:
This is in test so I don't personally have concerns making it public, but others might feel differently.

Reply:
No longer public.